api/routes/userRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const { userController } = require('../controllers');
+const { loginRequired } = require('../utils/auth');
+
+const userRouter = express.Router();
+
+userRouter.get('/mypage', loginRequired, userController.getUserInformation);
```

Comment:
/mypage 를 안 해도 충분할 것 같아요
```suggestion
userRouter.get('', loginRequired, userController.getUserInformation);
```

Reply:
엔드포인트 변경 및 테스트 완료하였습니다!